### DISCUSSION TITLE
32 bit version?

### DISCUSSION THREAD

**Discussion by Reggie521** [d2phap/ImageGlass#938] [General] [answered]
Is there still a 32 bit version of this program? I can't update to 8.0 because the only installer to download is x64.

**Comment by d2phap** [comment]
Hi @Reggie521 

Yes, it will come a bit later. Because 64-bit uses Magick.NET OpenMP which only supports 64 bit.
Thanks!

**Comment by d2phap** [comment, marked as answer]
Added 32-bit versions!
https://imageglass.org/download
